bitcoin-magazine/bitcoin-2024/making-bitcoin-more-private-with-cisa.md

```diff
@@ -184,110 +187,119 @@ So that's how I would encourage you to think about it from the start.
 I think the primary motivator was to keep Taproot manageable in terms of review effort and just keep the scope smaller.
 The only thing that I could see really in between the lines, what Craig already kind of mentioned is that if you just look at the pure fee savings numbers and the numbers of weight units that you save, if you just throw that over to somebody that it's for half aggregation in the single digits, then people are often a bit disappointed as a first reaction and you have to really discuss it and fill it a bit more with this understanding and I think that made it also have turned off some people and like why maybe also developers felt like this would be the easiest thing to chop off but that people wouldn't miss as much.
 
-Speaker 2: 00:15:53
+Craig Raw: 00:15:53
 
 Yeah so I mean I think you know in terms of the whatever pushback I have heard so far is what Fabian was saying that it doesn't do enough.
-You know, I think that there's a general perception that, you know, we can only ever push for one soft fork at a time, so all the soft forks have to compete to be the one, which is an interesting point of view.
+I think that there's a general perception that, you know, we can only ever push for one soft fork at a time, so all the soft forks have to compete to be the one, which is an interesting point of view.
 
-Speaker 3: 00:16:17
+Jameson Lopp: 00:16:17
 
-It's interesting because I was in the room in Hong Kong when Peter Wille announced that we were changing the flagging system so that we could do like 32 soft forks in parallel.
+It's interesting because I was in the room in Hong Kong when Pieter Wuille announced that we were changing the flagging system so that we could do like 32 soft forks in parallel.
 And of course, we have not yet taken advantage of that.
 
-Speaker 2: 00:16:33
+Craig Raw: 00:16:33
 
 Yeah, so we're in a very different space from that now.
-I think something to bear in mind, if you're thinking about it from that point of view, is that, and I heard Brandon Black actually, who was here earlier today talking about Opcad, is that I think Caesar is probably the most low-risk soft fork that we could imagine.
-And I'm talking here about the well-understood half signature aggregate form of it.
+I think something to bear in mind, if you're thinking about it from that point of view, is that, and I heard Brandon Black actually, who was here earlier today talking about Opcad, is that I think CISA is probably the most low-risk soft fork that we could imagine.
+And I'm talking here about the well-understood half-signature aggregate form of it.
 You know, it's really from a security model point of view, the cryptographers tell us that there is no risk effectively or very low risk to doing this.
 It really takes very well understood properties of Schnorr, which is really just adding these signatures up and uses that.
 
 ## The Limited Potential of Signature Compression
 
-Speaker 2: 00:17:18
```

Comment:
I'm keep repeating that you shouldn't remove speaker attribution after a chapter name

Reply:
I had this in mind but I think I did the opposite.